<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MyAppClass</class>
 <widget class="QMainWindow" name="MyAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>525</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>525</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>525</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MyApp</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="Table">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>280</width>
      <height>480</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>280</width>
      <height>480</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>280</width>
      <height>480</height>
     </size>
    </property>
    <row>
     <property name="text">
      <string>1</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>2</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>3</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>4</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>5</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>6</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>7</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>8</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>9</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>10</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>11</string>
     </property>
    </row>
    <row>
     <property name="text">
      <string>12</string>
     </property>
    </row>
    <column>
     <property name="text">
      <string>Дата</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Сумма</string>
     </property>
    </column>
   </widget>
   <widget class="QPushButton" name="Evaluate">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>330</y>
      <width>90</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Enter">
    <property name="geometry">
     <rect>
      <x>55</x>
      <y>140</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="maxLength">
     <number>32761</number>
    </property>
    <property name="placeholderText">
     <string>Введите сумму</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="kvartal1">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>190</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>I квартал</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="kvartal2">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>210</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>II квартал</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="kvartal3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>230</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>III квартал</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="kvartal4">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>250</y>
      <width>90</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IV квартал</string>
    </property>
   </widget>
   <widget class="QComboBox" name="Year">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>280</y>
      <width>70</width>
      <height>22</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>2021</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2022</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>2023</string>
     </property>
    </item>
   </widget>
   <widget class="QLabel" name="EmptyNum">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>120</y>
      <width>90</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Поле пусто!</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="EmptyKvart">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>170</y>
      <width>130</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Выберите квартал</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Incorrect">
    <property name="geometry">
     <rect>
      <x>65</x>
      <y>90</y>
      <width>140</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Некорректный ввод</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Result">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>440</y>
      <width>211</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Итого: </string>
    </property>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="MyApp.qrc"/>
 </resources>
 <connections/>
</ui>
